doc/appendix-de-kassensichv/procedural-documentation/dsfinv-k-generation.md

```diff
@@ -49,72 +49,72 @@ This chapter lists the data fields that are mandatory and have to be provided by
+| `POS_UST` | VAT | Decimal (5) | Automatically calculated and filled by fiskaltrust depending on `ftChargeItemCase` |
 
 
-##### File: Bonpos_Preisfindung (itemamounts.csv)
+#### File: Bonpos_Preisfindung (itemamounts.csv)
 
-Not supported.
+fiskaltrust does currently not support the file Bonpos_Preisfindung.
 
-According to the DSFinV-K specification, in the data field STK_BR in the Bonpos file, either the reduced amount is displayed immediately (and the origin of the amount in the file Bonpos_Preisfindung) or the reduction in charges is displayed as a separate item line with negative amounts (with correct tax assignment; see file Bonpos_USt). The GV_TYP "Rabatt" is available for the separate line. For the reduction in charges, the current version of the ft.Middleware expects a separate negative `ftChargeItem` with the corresponding `ftChargeItemCase` (e.g. `0x4445000000000031`) which mapps to the GV_TYP "Rabatt". ft does currently not support the file Bonpos_Preisfindung.
+According to the DSFinV-K specification, in the data field STK_BR in the Bonpos file, either the reduced amount is displayed immediately (and the origin of the amount in the file Bonpos_Preisfindung) or the reduction in charges is displayed as a separate line item with negative amounts (with correct tax assignment; see file Bonpos_USt). The GV_TYP "Rabatt" is available for the separate line. For the reduction in charges, the current version of the fiskaltrust.Middleware expects a separate negative `ftChargeItem` with the corresponding `ftChargeItemCase` (e.g. `0x4445000000000031`) which maps to the GV_TYP "Rabatt". 
 
 
-##### File: Bonpos_Zusatzinfo (subitems.csv)
+#### File: Bonpos_Zusatzinfo (subitems.csv)
 
-This table allows to detail the composition of sold combinations of goods (DE: Warenzusammenstellungen). They serve exclusively for explanation.
+This table allows to detail the composition of sold combinations of goods (DE: Warenzusammenstellungen). They serve exclusively for an explanation.
 
-This does not affect the basis of assessment for VAT. In the case of goods combinations with different tax rates, however, information is stored here which serves to control the distribution of the VAT assessment basis (example: Inhouse breakfast menu consisting of a coffee, a drink and buffet meal). In addition, orders deviating from the standard order can be taken into account to record the actual consumption of goods (example: gyros plate with chips instead of rice).
+This does not affect the basis of assessment for VAT. In the case of goods combinations with different tax rates, however, information is stored here which serves to control the distribution of the VAT assessment basis (for example, Inhouse breakfast menu consisting of a coffee, a drink and buffet meal). In addition, orders deviating from the standard order can be taken into account to record the actual consumption of goods (for example, gyros plate with chips instead of rice).
```

Comment:
NEVER rice with gyros, I am sorry, but there are lines which we _cannot_ cross :D :D  always french fries!

Reply:
;-)